smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftDependency.kt

```diff
@@ -14,13 +14,7 @@ class SwiftDependency(
     private val location: String,
     private val localPath: String,
     override var packageName: String,
-    private val distributionMethod: DistributionMethod,
 ) : Dependency {
-    enum class DistributionMethod {
-        SPR,
-        GIT,
-    }
-
```

Comment:
`DistributionMethod` is removed because now everything is presumed distributed by git.